### PR TITLE
Composite products branch

### PR DESCRIPTION
**Pull request by k1kolev**
Proposal for an article featuring composite products in ERP.net.

Updating toc to include it.

### CODE REVIEW

**Review by thristozova — COMMENTED** [review]
Let's make the wms part describe not only the levels but the overall processing.

I'm giving an example as a guideline of how I think it would be better, but of course, you can edit and embellish it as you see fit.

E.g.

Title ## Composite Products in @@name WMS
@@name WMS fully supports working with composite products. The module automatically breaks down the composite product requisitions into orders for their components.  
All components' availability can be tracked in real-time.
WMS introduces several **levels of control** which can be set for **proper and controlled** task execution when working with composite products….

**Comment by k1kolev** [comment]
Updated the article to include the new suggestions by @thristozova.

**Review by thristozova — APPROVED** [review]

File: features/logistics/composite-products.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ Many other businesses, like those trading desktop and server configurations, HVA
...
-The system provides all necessary means to fulfill all kinds of orders and tasks associated with handling and managing composite products, accompanied by their related documentation.  
+The system provides all necessary means to **fulfill all kinds of orders and tasks** associated with **handling and managing composite products**, accompanied by their related **documentation**.  
 
-WMS is capable of breaking down requisitions for composite products to generate new orders for executing operations with their components.  
+WMS is capable of **breaking down requisitions** for composite products to **generate new orders** for executing operations with their components.  
 
-Availabilities for all components of composite products can be tracked in real-time by the help of the built-in Warehouse Availabilities Report functionality.  
+**Availabilities for all components** of composite products can be **tracked in real-time** by the help of the built-in **Warehouse Availabilities Report** functionality.  
 
 ## Levels of Control for Composite Products in WMS of @@name
```
Comment (i-argentinski) [inline features/logistics/composite-products.md:27]: We need to at least list the names of the "levels of control".

File: features/logistics/composite-products.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Composite Products
+
+## Overview
+
+There are industries (like furniture and fixtures) where businesses are accustomed to storing, handling, and distributing **products composed of individual parts or components** that may be stored and manipulated **separately** from one another.
```
Comment (i-argentinski) [inline features/logistics/composite-products.md:5]: "are accustomed" - unnecessary and not very adequate here. Just remove it. "businesses are storing, handling, ...."

**Review by i-argentinski — CHANGES_REQUESTED** [review]

**Review by thristozova — APPROVED** [review]

File: features/logistics/composite-products.md (hunk 1 above)
Comment (k1kolev) [reply features/logistics/composite-products.md:27]: Levels of control are listed and short descriptions of their purposes are provided.

File: features/logistics/composite-products.md (hunk 2 above)
Comment (k1kolev) [reply features/logistics/composite-products.md]: Removed.

File: features/logistics/composite-products.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Composite Products
+
+## Overview
+
+There are industries (like furniture and fixtures) where businesses are accustomed to storing, handling, and distributing **products composed of individual parts or components** that may be stored and manipulated **separately** from one another. 
+Since these components comprise the final distributed and sold product, their full set is treated as a **single product** and must be managed accordingly.  
+
+@@name is armed with the **Composite Products** feature to **meet all needs** and help **overcome all challenges** when dealing with composite products.
```
[the remark this answers is not in the document]
Comment (k1kolev) [reply features/logistics/composite-products.md]: Replaced.

**Comment by k1kolev** [comment]
Fixed some typos on the latest commit.

**Review by i-argentinski — APPROVED** [review]

**Pull request merged**